frontend/www/js/omegaup/components/course/AddStudents.vue

```diff
@@ -8,7 +8,10 @@
         "
       >
         <div class="form-group">
-          <label>{{ T.wordsStudent }}</label>
+          <label>{{ T.wordsUserName }}</label>
```

Comment:
```suggestion
          <label>{{ T.username }}</label>
```

así se evita agregar una cadena de traducción más.